pkgs/applications/graphics/pixinsight/default.nix

```diff
@@ -0,0 +1,127 @@
+{ stdenv, lib, requireFile, wrapQtAppsHook, autoPatchelfHook, makeWrapper, unixtools, fakeroot
+, mime-types, libGL, libpulseaudio, alsa-lib, nss, gd, gst_all_1, nspr, expat, fontconfig
+, dbus, glib, zlib, openssl, libdrm, cups, avahi-compat, xorg, wayland, libudev0-shim
+# Qt 5 subpackages
+, qt5
+# libsForQt5 sub-packages
+, qt3d, mlt
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pixinsight";
+  version = "1.8.8-12";
+
+  src = requireFile rec {
```

Comment:
Could you add the `message` attribute, explaining where to get this file? For example, see https://github.com/NixOS/nixpkgs/blob/1b1fe46bb1f2d86cd47353c8d921f6c09677b294/pkgs/games/sm64ex/default.nix#L13

Might be nice to explain how to add it to the store as well.